### PR TITLE
Better NPM workflow variables

### PR DESCRIPTION
**Pull request by tphalp**
I moved the variables up in the yaml file so they are easier to see and change, which is necessary for other adapter creators.

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -23,15 +26,12 @@ jobs:
 
       - name: 'Dependencies and tests'
         run: |
-          # NOTE: Change this to the directory where your package lives
-          PACKAGE_DIR=packages/mx-adapter
```
Comment (tphalp) [inline .github/workflows/npm-publish.yml:27]: Old location

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -9,6 +9,9 @@ jobs:
   push_to_registry:
     name: 'Setup, and publish'
     runs-on: ubuntu-latest
+    env:
+      # NOTE: Change this to the directory where your package lives
+      PACKAGE_DIR: 'packages/mx-adapter'
```
Comment (tphalp) [inline .github/workflows/npm-publish.yml:14]: New location

File: .github/workflows/npm-version-check.yml [hunk 3]
```diff
@@ -6,6 +6,11 @@ jobs:
   push_to_registry:
     name: 'NPM Version Check'
     runs-on: ubuntu-latest
+    env:
+      # NOTE: Change this to the directory where your package lives
+      PACKAGE_DIR: 'packages/mx-adapter'
+      # NOTE: Change this to the name of your package on NPMJS.com
+      NPM_PACKAGE_NAME: '@ucp-npm/mx-adapter'
```
Comment (tphalp) [inline .github/workflows/npm-version-check.yml:13]: New location

File: .github/workflows/npm-version-check.yml [hunk 4]
```diff
@@ -20,17 +25,12 @@ jobs:
 
       - name: 'Check if version is published'
         run: |
-          # NOTE: Change this to the directory where your package lives
-          PACKAGE_DIR=packages/mx-adapter
-          # NOTE: Change this to the name of your package on NPMJS.com
-          NPM_PACKAGE_NAME=@ucp-npm/mx-adapter
```
Comment (tphalp) [inline .github/workflows/npm-version-check.yml:26]: Old location

File: packages/mx-adapter/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ucp-npm/mx-adapter",
-  "version": "0.0.15-alpha",
+  "version": "0.0.16-alpha",
```
Comment (tphalp) [inline packages/mx-adapter/package.json:3]: Update to publish new version

**Review by wesrisenmay-mx — APPROVED** [review]

**Pull request merged**